### PR TITLE
Series 1 - Blog 1 and 2

### PR DESCRIPTION
**Pull request by MarvinBertin**

### CODE REVIEW

File: content/blog/series_1_blog_1/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: "Introduction to Crypto, A Misunderstood Technology"
+description: "testing testing testing"
+date: 2021-06-28T20:39:12-10:00
```
Comment (ggydush) [inline content/blog/series_1_blog_1/index.md:4]: Make sure to update date before merging!

File: content/blog/series_1_blog_1/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+1. Cardano is based on fundamental research, using first principle reasoning and peer-reviewed research  <sup>[[1]](https://docs.cardano.org/introduction)</sup>. In fact, it is an outlier in the crypto space where heuristics-based designs and fast iteration is the norm.
+
+2. Cardano is currently the only cryptocurrency primarily focused on developing markets by promoting a global financial system focused on banking the unbanked. Here it believes blockchain technologies will have the largest impact where 31% of adults worldwide still do not have a bank account <sup>[[2]](https://globalfindex.worldbank.org)</sup>. And with a strong presence in Africa and large projects already deployed in Ethiopia <sup>[[3]](https://www.coindesk.com/ethiopian-education-minister-confirms-cardano-blockchain-partnership)</sup> and Tanzania <sup>[[4]](https://news.bitcoin.com/cardano-blockchain-infrastructure-selected-in-a-new-drive-to-bring-sustainable-internet-connectivity-to-zanzibar-and-tanzania)</sup>, Cardona is well positioned to succeed in this space.
```
Comment (ggydush) [inline content/blog/series_1_blog_1/index.md:43]: `Cardona` <-- either this is a new coin I don't know about, or it's a typo

File: content/blog/series_1_blog_2/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+</p>
+
+Even more interesting to highlight is the effect Bitcoin will have on the economics of currencies themselves. Today, currencies are no longer backed by collateral, like gold-standard which the USA went off in 1963 <sup>[[6]](https://www.investopedia.com/ask/answers/09/gold-standard.asp)</sup>, due to overspending in the Vietnam War. As a consequence, central banks have full control of the money supply and can print money in exchange for government debt. Take the impact of the COVID-19 pandemic for instance, the US government’s stimulus increased the USD money supply by a whopping 345% in just 12 months and subsequently pushed the national debt to 123% of the U.S. GDP <sup>[[7]](https://fred.stlouisfed.org/series/GFDEGDQ188S)</sup>. While in the short term, the stimulus has greatly benefited many American households in need during the pandemic, it is unclear what will be the long term adverse effects on the economy over the next decade. In fact, this trend is here to stay and marks a turning point in US monetary policy. It even has a name, Modern Monetary Theory (MMT) <sup>[[8]](https://www.investopedia.com/modern-monetary-theory-mmt-4588060)</sup>.
```
Comment (ggydush) [inline content/blog/series_1_blog_2/index.md:53]: like **the** gold-standard which the USA went off in 1963 due to the overspending in the Vietnam war.

Pretty sure you can remove comma before due

File: content/blog/series_1_blog_2/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  <p align="center"><em>Total US Public Debt as Percentage of GDP</em></p>
+</p>
+
+> **"[...] global debts are at record highs and world nations are realizing they may be at the brink of another financial crisis similar to 2008 or maybe worse."**
+
+In fact, the transition may happen sooner than we think. The fastest adoption of Bitcoin has been seen in developing nations, such as Venezuela where its citizens are witnessing the collapse of their local currency through hyperinflation. As of June of 2021, El Savador is the first nation to officially accept Bitcoin as legal tender, with other nations already ready to follow suit. The cat is out of the bag and it is here to stay.
+
+
+## What are Bitcoin’s Shortcomings?
+Bitcoin introduced the world to a groundbreaking new approach to money. Fast Forward to today, institutional investors are now pouring billions of dollars into it and nations around the world, whether they like it or not, are forced to acknowledge its significance. However Bitcoin isn’t perfect and suffers from systemic issues that may jeopardize its long term success.
```
Comment (ggydush) [inline content/blog/series_1_blog_2/index.md:73]: Fast forward <- forward should be lower case

File: content/blog/series_1_blog_2/index.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
...s capacity to adapt and improve over time. Bitcoin, like most other cryptocurrencies, is a public and decentralized organization with no owner nor official hierarchical power structure, therefore the lack of a governing body means that any decision comes at the risk of fragmenting the network and its community. Ironically, this lack of adaptability could, over time, render Bitcoin obsolete before it even becomes mainstream. Most improvement proposals get stunted in endless debates between the Bitcoin developers and Bitcoin miners. In 2017, Bitcoin’s community went to war over a seemingly small change to the protocol related to the block size <sup>[[13]](https://coingeek.com/the-war-on-bitcoin/)</sup>. The disputes escalated and culminated in the eventual split of Bitcoin into three separate cryptocurrencies (Bitcoin, Bitcoin Cash and Bitcoin SV). This event is now referred to as the “Bitcoin Civil War” and since then no other significant changes have been introduced to the network. One way other cryptocurrency handle this limitation is by designing an “on-chain governance” system, in which any protocol decision is made by the community through a secure and democratic voting system.
```
Comment (ggydush) [inline content/blog/series_1_blog_2/index.md:83]: One way other cyptocurrency -> One way other **cryptocurrencies**

**Review by ggydush — APPROVED** [review]
Looks good to me! It would be awesome if we could get the next 2 blogs in tomorrow! Ideally we'd have this whole thing released at the start, especially because the point mentioned early on is "why we believe cardano is the only true 3rd generation crpyto"

File: content/blog/series_1_blog_1/index.md (hunk 1 above)
Comment (MarvinBertin) [reply content/blog/series_1_blog_1/index.md:4]: Can I skip adding the time? Just the ISO data format

**Comment by MarvinBertin** [comment]
I will work on blog 3 today (proof read, link sources add images, etc.). For Blog 4 I don't think I will have time today, bit definitely by next week

**Comment by MarvinBertin** [comment]
I still need to add a description and blog thumbnail images

**Pull request merged**